fictadvisor-api/src/v2/api/services/UserService.ts

```diff
@@ -263,9 +264,15 @@ export class UserService {
 
   async requestNewGroup (id: string, { groupId, isCaptain }: GroupRequestDTO) {
     const student = await this.studentRepository.findById(id);
-    if (student.state === State.APPROVED) {
+
+    switch (student.state) {
+    case State.APPROVED: {
```

Comment:
Add indentation for case blocks

Reply:
I tried to do this, but eslint highlights it as error